docs/Researcher/scheduling/GPU-time-slicing-scheduler.md

```diff
@@ -0,0 +1,85 @@
+---
+title: GPU Time Slicing Scheduler
+summary: This article describes the trainings feature. This feature provides a wizard like experience to submit training jobs.
+authors:
+    - Jason Novich
+    - Hagay Sharon
+date: 2023-Oct-25
+---
+
+Run:ai supports simultaneous submission of multiple workloads to a single GPU when using fractional GPU workloads. This is achieved by slicing the GPU memory between the different workloads according to the requested GPU fraction, and by using NVidia's GPU orchestrator to share the GPU compute runtime. Run:ai ensures each workload receives the exact share of the GPU memory (= gpu\_memory \* requested), while the NVidia GPU orchestrator splits the GPU runtime evenly between the different workloads running on that GPU. For example, when there are 2 workloads running on the same GPU, each will get 50% of the GPU compute runtime, even if one workload requested 25% of the GPU memory, and the other workload requests 75% of the GPU memory.
+
+## New Time-slicing scheduler by Run:ai
+
+To provide customers with predictable and accurate GPU compute resources scheduling, Run:ai is introducing a new feature called Time-slicing GPU scheduler which adds **fractional compute** capabilities on top of other existing Run:ai **memory fractions** capabilities. Unlike the default NVIDIA GPU orchestrator which doesn’t provide the ability to split or limit the runtime of each workload, Run:ai created a new mechanism that gives each workload **exclusive** access to the full GPU for a **limited** amount of time ([lease time](#timeslicing-plan-and-lease-times)) in each scheduling cycle ([plan time](#timeslicing-plan-and-lease-times)). This cycle repeats itself for the lifetime of the workload.
+
+Using the GPU runtime this way guarantees a workload is granted its requested GPU compute resources proportionally to its requested GPU fraction.
+
+Run:ai offers two new Time-slicing modes:
+
+1. **Strict**&mdash;each workload gets its **precise** GPU compute fraction, which equals to its requested GPU (memory) fraction. In terms of official Kubernetes resource specification, this means:
+
+```
+gpu-compute-request = gpu-compute-limit = gpu-(memory-)fraction
+```
+
+2. **Fair**&mdash;each workload is guaranteed at least its GPU compute fraction, but at the same time can also use additional GPU runtime compute slices that are not used by other idle workloads. Those excess time slices are divided equally between all workloads running on that GPU (after each got at least its requested GPU compute fraction). In terms of official Kubernetes resource specification, this means:
+
+```
+gpu-compute-request = gpu-(memory-)fraction
+
+ or
```

Comment:
Not 'or', it's an 'and' logical operation, you get at least the request and you can get up to the whole GPU. Mathematically this can be represented as